pages/themes/newest.js

```diff
@@ -1,14 +1,23 @@
 import PluginThemeShowcase from 'components/plugin-theme-showcase'
 import allPlugins from 'plugins'
 
-export default () => {
-  const plugins = allPlugins
+export default function ThemeNewestPage({ themes }) {
+  return (
+    <PluginThemeShowcase plugins={themes} variant="theme" filter="newest" />
+  )
+}
+
+export function getStaticProps() {
+  // get all featured plugins then get the preview image's relative src
+  const themes = allPlugins
     .filter((p) => p.type === 'theme')
     .sort((a, b) =>
       a.dateAdded < b.dateAdded ? 1 : a.dateAdded > b.dateAdded ? -1 : 0
```

Comment:
Does this need the preview image?
```
      preview: getPluginPreviewImage(p.name),
```

Reply:
The comment there is misleading but the code is correct, this page is the "list" view for the newest themes, there are no preview images present here. Removed the comment.